runtime/compiler/build/toolcfg/zos-xlc/common.mk

```diff
@@ -73,6 +73,7 @@ CX_DEFINES+=\
     $(HOST_DEFINES) \
     $(TARGET_DEFINES) \
     J9ZOS390 \
+    _AE_BIMODAL \
```

Comment:
I think this is more closely connected to `IBM_ATOE` and should go after line 112 where it is defined.